src/loaders/index.ts

```diff
@@ -0,0 +1,57 @@
+import { LoadSpec, loadSpecToString, IVolumeLoader, VolumeDims } from "./IVolumeLoader";
+
+import { OMEZarrLoader } from "./OmeZarrLoader";
+import { JsonImageInfoLoader } from "./JsonImageInfoLoader";
+import { TiffLoader } from "./TiffLoader";
+import VolumeCache from "../VolumeCache";
+
+const enum VolumeFileFormat {
+  ZARR = "zarr",
+  JSON = "json",
+  TIFF = "tiff",
+}
+
+type CreateZarrLoaderOptions = { path: string; cache?: VolumeCache; scene?: number };
+type CreateJsonLoaderOptions = { path: string | string[]; cache?: VolumeCache };
+type CreateTiffLoaderOptions = { path: string };
```

Comment:
For simplicity, why not just make one generic union type that has path, cache, and scene? (Do we even use scene? I think not, for zarr?).   Then you can just pass LoaderOptions right in to the factory function

Reply:
Scene is added as an option to `OMEZarrLoader` by this PR. My thought was, in a loaders-as-handles world it makes more sense to pick a scene at construction time rather than allow scenes with potentially different dimensions to be selected and reselected at load time.

Reply:
I agree that scene should be selected at load time.  But I'm semi-proposing that we remove scene for now, as it is a higher-level detail on how zarr data is organized, and we really don't have a good "multi-scene" example to work with, if I remember correctly.  At some level a scene is just a whole zarr file.